### PR TITLE
Add gif or gif

### PR DESCRIPTION
**Pull request by ionrock**
We'll never truly know!

### CODE REVIEW

File: giforgif/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: It's pronounced GIF!
+---
+
+# It is gif or gif?
+
+<img src="https://images.duckduckgo.com/iu/?u=http%3A%2F%2Fimg.pandawhale.com%2F90837-what-is-GIF-JIF-jeopardy-YES-I-sZL0.gif&f=1" />
```
Comment (moutons) [inline giforgif/index.md]: Snag the image and host in the giforgif directory, pop. I'm paying the S3 bill so 💸

Comment (filler) [reply giforgif/index.md]: dat :cloud: :moneybag: tho

**Comment by filler** [comment]
Fix it, @ionrock! Fix it!

**Comment by moutons** [comment]
pls for to fix @ionrock

**Comment by gwaldo** [comment]
You know... as in the "J" sound in "Graphical"

;-)

**Comment by filler** [comment]
rubbyjems ...

**Pull request merged**